packages/webchat/src/main.tsx

```diff
@@ -196,7 +192,6 @@ class Web extends React.Component<MainProps> {
           this.props.displayWidgetView ? this.props.showChat!() : this.props.hideChat!()
           trackWebchatState('toggle')
         } else if (type === 'message') {
-          trackMessage('sent')
```

Comment:
Any particular reason why you removed this?

Reply:
because it's tracked in the sendMessage function as well ==> tracked twice